README.md

```diff
@@ -70,8 +70,9 @@ def users_table() -> list[AnyComponent]:
         c.Page(  # Page provides a basic container for components
             components=[
                 c.Heading(text='Users', level=2),  # renders `<h2>Users</h2>`
-                c.Table[User](  # c.Table is a generic component parameterized with the model used for rows
+                c.Table(
                     data=users,
+                    data_model=User,
```

Comment:
You don't need this unless there's a chance that `data` is empty, in this case we should use the most minimal example.
```suggestion
```